src/roaring.c

```diff
@@ -81,6 +82,35 @@ roaring_bitmap_t *roaring_bitmap_create_with_capacity(uint32_t cap) {
     return ans;
 }
 
+
+void roaring_bitmap_add_ordered(roaring_bitmap_t *r, size_t n_args, const uint32_t *vals) {
```

Comment:
Maybe you should make `roaring_bitmap_add_ordered` return an error code, e.g., `1` if the values are ordered and `0` if they are not.

Reply:
roaring_bitmap_writer_add() has an assert() for this

Reply:
I saw, but this is a problem since we cannot "catch" it. In case of invalid input, the program will crash, so the user has to check that the operation is valid beforehand.

This is similar to [`roaring_bitmap_select`](https://github.com/RoaringBitmap/CRoaring/blob/master/include/roaring/roaring.h#L515-L521): this function returns true when the operation is valid, false otherwise.

Reply:
Got it! I made it return true indicating success.